packages/visx-brush/src/BaseBrush.tsx

```diff
@@ -405,6 +405,19 @@ export default class BaseBrush extends React.Component<BaseBrushProps, BaseBrush
                 if (onMouseUp) onMouseUp(event);
                 dragEnd(event);
               }}
+              onTouchStart={(event: MouseHandlerEvent) => {
```

Comment:
on thought I have here for DRY-ness is to update all of these to `PointerEvent` handlers, which are a combination of `MouseEvent`s + `TouchEvent`s introduced in [`react@16.4`](https://reactjs.org/blog/2018/05/23/react-v-16-4.html).  so `onMouseDown => onPointerDown`, `onMouseLeave => onPointerLeave`, `onMouseMove => onPointerMove`. I think that `MouseHandlerEvent` will also need to be updated to support `React.PointerEvent`. 

To avoid breaking changes we could could keep the props as `onMouseXXX`, but could consider also adding `onPointerXXX` handlers to match, and then deprecate `onMouseXXX` in a future `2.0` release. This would also require that we bump the `react` `peerDependency` in `@visx/brush` to `^16.4.0`, which is theoretically breaking but I just checked `@visx/drag` (which brush relies on) and it already requires `^16.8.0` so I think technically `@visx/brush` should already actually be `^16.8.0`.

cc @hshoff